OpenTabletDriver.Configurations/Configurations/RobotPen/T9A.json

```
@@ -0,0 +1,41 @@
+{
+  "Name": "RobotPen T9A",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 300.0,
+      "Height": 210.0,
+      "MaxX": 22784.0,
+      "MaxY": 16896.0
```

The ratios for Width/Height and MaxX/MaxY do not match. These also doesn't represent a sensible LPI/LPmm, but it's near 2000 LPI.

Did you mean:
```suggestion
      "Width": 289.36,
      "Height": 214.58,
      "MaxX": 22784.0,
      "MaxY": 16896.0
```